toolset/utils/docker_helper.py

```diff
@@ -131,8 +134,13 @@ def build(self, test, build_log_dir=os.devnull):
                 log_prefix=log_prefix,
                 path=test.directory,
                 dockerfile=test_docker_file,
-                tag="techempower/tfb.test.%s" % test_docker_file.replace(
-                    ".dockerfile", ""))
```

Comment:
lol, what terrible code I wrote. "Make `test_docker_file` by combining `test.name` and ".dockerfile", then remove ".dockerfile" to find the name of the test." 👍